java/src/jmri/jmrit/logixng/tools/swing/ImportLogixFrame.java

```diff
@@ -178,7 +179,10 @@ private void doImport() {
         List<Logix> logixs = new ArrayList<>();
         if (_whichLogix_All.isSelected()) {
             for (Logix logix : InstanceManager.getDefault(LogixManager.class).getNamedBeanSet()) {
-                boolean isSystemLogix = logix.getSystemName().equals(SYSLOGIX) || logix.getSystemName().equals(INITLOGIX);
+                boolean isSystemLogix =
+                        logix.getSystemName().equals(SYSLOGIX) ||
+                        logix.getSystemName().contains(RTXLOGIX) ||
+                        logix.getSystemName().contains(USSLOGIX);
```

Comment:
As I said above, I suggest you use `logix.getSystemName().startsWith` instead of `logix.getSystemName().contains`.

Reply:
See above about the system name prefix.